internal/fileutils/fileutils.go

```diff
@@ -681,6 +696,56 @@ func WriteTempFileToDir(dir, pattern string, data []byte, perm os.FileMode) (str
 	return f.Name(), nil
 }
 
+func CopyFilesFromAssets(assetDir, dst string) error {
```

Comment:
I'd like to keep `fileutils` unopinionated about the rest of our codebase. Granted this is a ship that has sailed, but I'd like to get back to that point eventually so we can treat this as a standalone package that we might even open source.

Instead of giving this package opinions about assets, we could give the asset package a way of interacting with it via a struct rather than on the package top level. Then just turn this function into one that uses an interface with `ReadDir()`.